src/style.css

```diff
@@ -1,3 +1,72 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+
+:root {
+  --color: #00acdc;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
Avoid using the '*' selector as it impacts performance. Set styles only for elements that require them. Try to use more specific selectors that target only the elements you want to style.